napari/_qt/qt_resources/styles/02_custom.qss

```diff
@@ -182,7 +182,7 @@ QtLayerWidget > QLineEdit {
   border: none;
   border-radius: 2px;
   padding: 2px;
-  font-size: 14px;
+  font-size: {{ increase(font_size, 2) }};
```

Comment:
Sorry, stupid question maybe but this is/was 14px, while the other places were in `pt`, does that change anything ?

It's here and in a bunch of places, so I don't know if it's relevant or not.

Reply:
We should everywhere use pt. `px` is based on physical screen pixel size, and `pt` base on dpi of the screen.

Reply:
> We should everywhere use pt. px is based on physical screen pixel size, and pt base on dpi of the screen.

Sure I know the pt/px/em/ex units, let me rephrase.

Is replacing 14px by 14pt the same for qt or do we need to have a correcting factor. Or maybe it was just a typo and was 14pt what was originally meant.

It also relatively easy to scan through a file and not realize that some units are pt and px.

Reply:
Checking this, I think there is a factor we will need to take into account since `pt` vs `px` is not a one on one equivalence. 14pt are actually ~18px (online converter to check: https://pixelsconverter.com/pt-to-px). Also, when checking this locally on Windows I can see that some of the aplication text has increased in size which probably is due to the changing to use pt but keeping the same px value:

### This PR:

![imagen](https://github.com/napari/napari/assets/16781833/9ed89df4-120d-414f-81fb-54e0bed263f9)

![imagen](https://github.com/napari/napari/assets/16781833/b35d0428-daff-4f26-bb73-e058f1ea92a6)

### main

![imagen](https://github.com/napari/napari/assets/16781833/35b61eab-7194-4581-b549-f5b8fcdb2f2a)

![imagen](https://github.com/napari/napari/assets/16781833/5817a105-64c5-492f-88fe-b5bea49c1af9)

Reply:
I think the px/pts relation also depends on the `ppi` resolution of the display, which differs between windows and macOS, so that's probably the effect.

Reply:
Oh I see, so should we create some function that helps with the conversion from px to pt and that queries the current value of `ppi`? 

Or maybe define some px to pt baseline equivalence to use in order to replace the px values that are being changed here and that work for Windows, MacOs and Linux? Looking around the PR seems like the values in px where:

* 10px
* 11px
* 12px
* 14px 
* 15px
* 16px
* 20px

Reply:
Also, should the default `font-size` be set to 12pt as it is right now (and I guess though as equivalent of 12px) or a different value? 🤔

Reply:
So, after playing a little bit with the base theme value, seems like using 13pt on macOS and 9pt on Windows and Linux preserves how things look on main. However extra checking from others is appreciated!